app/terms-of-service/page.tsx

```diff
@@ -0,0 +1,155 @@
+import { NextPage } from 'next';
+
+const PrivacyPolicyPage: NextPage = () => (
+  <div className='flex-1 w-screen flex flex-col justify-center items-center py-20'>
+    <div className='flex flex-col w-3/4 items-center'>
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        Terms of Service
+      </h2>
+      <span className='border border-b w-full border-accent' />
+    </div>
+    <br />
+    <div className='w-3/4'>
+      <h2 className='text-xl font-semibold mb-1'>
+        Last Updated: 01/04/2024
+      </h2>
+
+      <br />
+      <p>
+        These Terms of Service (&quot;Terms&quot;) govern your use of the Rental Review website
+        and mobile application (collectively, the &quot;Platform&quot;). By accessing or using
+        the Platform, you agree to be bound by these Terms.
+      </p>
+
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        1. User Eligibility
+      </h2>
+      <p className='ml-10'>
+        You must be at least 18 years old to use the Platform. By using the Platform, you
+        represent and warrant that you are of legal age to form a binding contract with us.
+      </p>
+
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        2. Account Registration
+      </h2>
+      <p className='ml-10'>
+        You may need to create an account to access certain features of the Platform. When
+        registering for an account, you agree to provide accurate, current, and complete
+        information about yourself.
+      </p>
+
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        3. User Conduct
+      </h2>
+      <li className='ml-10 w-fit'>
+        You agree to use the Platform for lawful purposes only and to comply with all
+        applicable laws and regulations.
+      </li>
+      <li className='ml-10 w-fit'>
+        You may not engage in any conduct that violates the rights of others, including but
+        not limited to infringing intellectual property rights, harassing or defaming others,
+        or distributing spam or malware.
+      </li>
+
+      <br />
+      <h2 className='text-2xl font-semibold mb-1 w-fit text-accent'>
+        4. Reviews and Ratings
```

Comment:
If we're going to allow image uploads, we probably want a notice about only allowing relevant and appropriate images to be uploaded. also point to our moderation system